packages/bff-types-generated/queries/dialog.fragment.graphql

```diff
@@ -39,6 +39,33 @@ fragment SearchDialogFields on SearchDialog {
   systemLabel
 }
 
+fragment SearchAutocompleteDialogFields on SearchDialog {
```

Comment:
Er alle disse feltene nødvendig for å vise autocomplete items in søkefeltet når vi bare bruker id, tittel og beskrivelse? Kanskje noe mer jeg har glemt.